### PR TITLE
Added Telnet Modules Docs

### PR DESCRIPTION
**Pull request by Yashvendra**
Added `auxiliary/scanner/telnet/telnet_login` and `auxiliary/scanner/telnet/telnet_version` modules documentation.

### CODE REVIEW

**Comment by jrobles-r7** [comment]
## Release Notes

Add module documentation for `auxiliary/scanner/telnet/telnet_login` and `auxiliary/scanner/telnet/telnet_version`.

File: documentation/modules/auxiliary/scanner/telnet/telnet_login.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+## Description
+This module will test a telnet login with a list of provided credentials on a range of machines and report successful logins.
+
+## Verification Steps
+
+1. Do: ```use auxiliary/scanner/telnet/telnet_login```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```set THREADS [NUMBER OF THREADS]```
+4. Do: ```set USER_FILE [USERNAME FILE]```
+5. Do: ```set PASS_FILE[PASSWORD FILE]```
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/telnet/telnet_login.md]: missing space after FILE

File: documentation/modules/auxiliary/scanner/telnet/telnet_login.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+6. Do: ```run```
+
+## Options
+
+This auxiliary module allows you to pass credentials in a number of ways. You can specifically set a username and password, you can pass a list of usernames and a list of passwords for it to iterate through, or you can provide a file that contains usernames and passwords separated by a space.
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/telnet/telnet_login.md]: This isn't needed and is more or less a duplicate of the description section

File: documentation/modules/auxiliary/scanner/telnet/telnet_login.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Only one password to authenticate with. 
+
+### USERPASS_FILE 
+
+File containing username and passwords separated by space, one pair one line.
+
+### USER_FILE
+
+File containing username one per line.
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/telnet/telnet_login.md]: username**s**

File: documentation/modules/auxiliary/scanner/telnet/telnet_version.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+## Description
+This module will scan a range of machines and prints any telnet servers that are running on it.
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/telnet/telnet_version.md]: prints **The banner, usually containing the version, of any** telnet

**Pull request merged**